.github/workflows/continuous-integration.yml

```diff
@@ -52,6 +54,8 @@ jobs:
 
       - name: "Install dependencies with Composer"
         uses: "ramsey/composer-install@v1"
+        with:
+          composer-options: "--ignore-platform-req=php+"
```

Comment:
We need to ignore the upper boundary of PHP version constraints. Otherwise, we cannot install Prophecy (as a dependency of PHPUnit) on PHP 8.2.

Reply:
Didn't know about that `+` syntax :bulb: